frontend/components/CourseFieldSelector.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="courses">
+    <select class="selector" v-model="myChoice" name="Kierunek">
+      <option :value="null" disabled hidden>Kierunek</option>
+      <option value="Informatyka">Informatyka</option>
+      <option value="Grafika">Grafika</option>
+      <option value="Ekonomia">Ekonomia</option>
+    </select>
```

Comment:
As above, `data` and `v-for`

Reply:
These are just test data, but I changed to v-for usage.